src/server/game/DataStores/DB2Structure.h

```diff
@@ -2049,6 +2049,21 @@ struct ScalingStatDistributionEntry
     uint32 MaxLevel;
 };
 
+struct SceneScriptEntry
+{
+    uint32 Entry;
+    char const* Name;
```

Comment:
This will not work properly, by default all strings are localized in DB2 and you must use LocalizedString* for this. If you know for sure that this will never contain a value in a different language (for example lua string references or entire lua script in this file) then you can use `char const*` BUT you must also change the metadata string from `s` to `S`